File: Practice/21/C++/21/21.cpp
#include <iostream>
using namespace std;

double bmi(double weight, double height) {
    double BMI;
    height = height / 100;
    BMI = weight / (height * height);
    return BMI;
}

void printBMI(double BMI){
    double bmi(double weight, double height);
    if (BMI < 18.5) cout << "Underweight";
    else if (BMI >= 18.5 && BMI < 25.0) cout << "Normal weight";
    else if (BMI >= 25.0 && BMI < 30.0) cout << "Overweight";
    else if (30.0 <= BMI) cout << "Obesity";
}
int main()
{
    double weight, height;
    cout << "Put your weight and height\n";
    cin >> weight >> height;
    printBMI(bmi(weight, height));

    return 0;
}

File: Practice/21/Python/21.py

def bmi(weight:float, height:float):
    height = height / 100
    bmi = weight / (height * height)
    return bmi

def print_bmi(bmi:float):
    if (bmi < 18.5): print("Underweight")
    elif (bmi >= 18.5 and bmi < 25.0): print("Normal weight")
    elif (bmi >= 25.0 and bmi < 30.0): print("Overweight")
    elif (30.0 <= bmi): print("Obesity")

weight,height = map(float, input("Put your weight and height\n").split())
print_bmi(bmi(weight, height))


